openedx/core/djangoapps/schedules/management/commands/tests/test_send_course_update.py

```diff
@@ -1,15 +1,20 @@
+import ddt
+from edx_ace.utils.date import serialize
 from mock import patch
 from unittest import skipUnless
 
 from django.conf import settings
 
 from openedx.core.djangoapps.schedules import resolvers, tasks
 from openedx.core.djangoapps.schedules.management.commands import send_course_update as nudge
-from openedx.core.djangoapps.schedules.management.commands.tests.send_email_base import ScheduleSendEmailTestBase
+from openedx.core.djangoapps.schedules.management.commands.tests.send_email_base import ScheduleSendEmailTestBase, \
+    ExperienceTest
```

Comment:
Nit: instead of an ending slash, can we list the imports within parens?

Reply:
Done